### PR TITLE
[ZEPPELIN-1747] Fix Korean notename input problem

### PR DESCRIPTION
**Pull request by marchpig**
### What is this PR for?
Korean notename is incorrectly typed on Firefox.
This PR fixes the issue by changing placeholder attribute of the input field.
Getting the placeholder text from `noteName()` is unnecessary because the text is visible only when the notename is blank.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1747

### How should this be tested?
Type Korean notename in action bar using Firefox browser.

### Screenshots (if appropriate)
korean-notename-issue

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by tae-jun** [comment]
I tested on Firefox and the error is gone. LGTM 👍 

image

And I guess the CI failure is unrelated. Could you re-trigger CI?

**Comment by marchpig** [comment]
@tae-jun Thanks for your quick reply.
I triggered CI again.

**Comment by Leemoonsoo** [comment]
Thanks @marchpig for the fix.
Thanks @tae-jun for the review.

LGTM and merge to master if there're no more comments.

File: zeppelin-web/src/app/notebook/notebook-actionBar.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <h3>
     <div style="float: left; width: auto; max-width: 40%"
       ng-controller="ElasticInputCtrl as input">
-      <input type="text" pu-elastic-input class="form-control2" placeholder="{{noteName(note)}}" style="min-width: 0px; max-width: 95%;"
+      <input type="text" pu-elastic-input class="form-control2" placeholder="Note {{note.id}}" style="min-width: 0px; max-width: 95%;"
```
Comment (cuspymd) [inline zeppelin-web/src/app/notebook/notebook-actionBar.html]: I understood this fix. 
But displaying a note id in the placeholder seems to be weird also, isn't it? 
The placeholder is just a simple value or a short description of the expected input.

Comment (cuspymd) [inline zeppelin-web/src/app/notebook/notebook-actionBar.html]: I understood this fix. 
But displaying a note id in the placeholder seems to be weird also, isn't it? 
The placeholder is just a simple value or a short description of the expected input.

Comment (marchpig) [inline zeppelin-web/src/app/notebook/notebook-actionBar.html]: @cuspymd Thanks for your comment! As you can see, I just wanted to fix the Korean input problem without any changes to the existing behavior. However, I'm not sure what value would be appropriate for this placeholder.

Comment (cuspymd) [inline zeppelin-web/src/app/notebook/notebook-actionBar.html]: @marchpig A watch is not cheap resource in AnguarJS. 
Let's erase meaningless interpolation '{{..}}'.
How about just "New Name" ?

Comment (marchpig) [inline zeppelin-web/src/app/notebook/notebook-actionBar.html]: Thanks @cuspymd for your suggestion. "New name" seems to be appropriate!

**Comment by Leemoonsoo** [comment]
Thanks @cuspymd for providing review.
LGTM and merge to master if there're no more comments.

**Pull request closed without merging**